.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+## PR Type
+What kind of change does this PR introduce?
+
+<!-- Please check the one that applies to this PR using "x". -->
+```
+[ ] Bugfix
+[ ] Feature
+[ ] Code style update (formatting)
+[ ] Refactoring (no functional changes, no api changes)
+[ ] Build or CI related changes
+[ ] Documentation content changes
+[ ] Sample app changes
+[ ] Other... Please describe:
+```
+
+
+## What is the current behavior?
+<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
+Issue Number: N/A
+
+
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tested code with SDK 14393 and SDK 15063
+- [ ] Docs have been added / updated (for bug fixes / features)
+- [ ] Sample in sample app has been added / updated (for bug fixes / features)
+- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
+
+
+## What is the new behavior?
+
+
+## Does this PR introduce a breaking change?
+```
+[ ] Yes
+[ ] No
+```
+
+<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->
```

Comment:
Perhaps also make a footnote here that breaking changes are quite likely to get rejected.

Reply:
Added